File: observe/dataservice/data-service/src/main/java/ai/whylabs/dataservice/structures/KeyValueTag.java
package ai.whylabs.dataservice.structures;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
public class KeyValueTag {
  @Schema(required = true)
  private String key;

  @Schema(required = true)
  private String value;
}

File: observe/api-service/service/src/main/kotlin/ai/whylabs/songbird/v0/models/KeyValueTag.kt
package ai.whylabs.songbird.v0.models

import io.swagger.v3.oas.annotations.media.Schema
import javax.validation.constraints.NotBlank

@Schema(description = "A key value tag")
data class KeyValueTag(
    @field:NotBlank val key: String,
    @field:NotBlank val value: String,
)
